### ISSUE TITLE
Can't connect to a running HTTP (non SSL) chainweb node

### ISSUE THREAD

**Issue by CryptoPascal31** [kadena-io/kda-tool#16]
My chainweb node is running on my local network
   - host: kadena-2
   - Pact (service) port is 1848 in HTTP (non SSL).
   
It looks like kda-tool is unable to connect to a non SSL port:

```
pascal@kadena-4:/mnt/common/test_kda_tool$ kda poll tx.json -n kadena-2:1848
kda: HttpExceptionRequest Request {
  host                 = "kadena-2"
  port                 = 1848
  secure               = True
  requestHeaders       = []
  path                 = "/info"
  queryString          = ""
  method               = "GET"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}
 ConnectionTimeout

pascal@kadena-4:/mnt/common/test_kda_tool$ curl http://kadena-2:1848/info
{"nodeNumberOfChains":20,"nodeApiVersion":"0.0","nodeChains":["12","13","14","15","8","9","10","11","4","5","6","7","0","16","1","17","2","18","3","19"],"nodeVersion":"testnet04","nodeGraphHistory":[[332604,[[12,[13,11,2]],[13,[12,14,3]],[14,[13,15,4]],[15,[14,0,16]],[8,[5,6,3]],[9,[4,6,7]],[10,[11,0,19]],[11,[12,10,1]],[4,[14,9,19]],[5,[8,7,0]],[6,[8,9,1]],[7,[9,5,2]],[0,[15,10,5]],[16,[15,1,17]],[1,[11,6,16]],[17,[16,2,18]],[2,[12,7,17]],[18,[17,3,19]],[3,[13,8,18]],[19,[10,4,18]]]],[0,[[8,[9,7,3]],[9,[8,4,5]],[4,[9,1,2]],[5,[9,6,0]],[6,[5,7,1]],[7,[8,6,2]],[0,[5,2,3]],[1,[4,6,3]],[2,[4,7,0]],[3,[8,0,1]]]]]}
 
```

However the tool is working with an https node (like api.testnet.chainweb.com):

```
pascal@kadena-4:/mnt/common/test_kda_tool$ kda poll tx.json -n api.testnet.chainweb.com
{"api.testnet.chainweb.com":[{"body":{"abIXD7WIg6YrOOb6YlEjhSWIUVHCslWSmZnjceS5a9A":{"reqKey":"abIXD7WIg6YrOOb6YlEjhSWIUVHCslWSmZnjceS5a9A","result":{"status":"success","data":"Keyset defined"},"metaData":{"blockTime":1673102579984445,"blockHeight":2883949,"blockHash":"KAlHK8yKuq3jltnRoLeo3NyxDDY1NnivkG7p7ykFKcE","prevBlockHash":"bFOMZ944CLd2ZODxUpWnED3HHr7pgJrANEXjmDrRsf4"},"gas":383,"logs":"XKD-QmNwDCkxJqc1mNJNKbDJ_oAbgm3q35FawOL1tGQ","events":[{"name":"TRANSFER","params":["k:4ba39bd19b9f3ecae0f03375f9bbe370b66e0101f1ffce5dbff0f3e9f9290967","k:db776793be0fcf8e76c75bdb35a36e67f298111dc6145c66693b0133192e2616",3.83e-6],"moduleHash":"rE7DU8jlQL9x_MPYuniZJf5ICBTAEHAIFQCB4blofP4","module":{"name":"coin","namespace":null}}],"continuation":null,"txId":2610711}},"statusMsg":"OK","statusCode":200}]}
```

Prefixing the host with `http://` does not help either:
```
pascal@kadena-4:/mnt/common/test_kda_tool$ kda poll tx.json -n http://kadena-2:1848
option -n: input does not start with a digit

Usage: kda poll [TX_FILE] [-n|--node NODE]

  Poll command results with a node's /poll endpoint
```

**Comment by mightybyte** [comment]
This issue should be fixed by PR #20 which allows you to specify http or https for your node.

**Comment by CryptoPascal31** [comment]
I've just compiled and tested it.
I confirm that it works now with an HTTP node. Thank you 
=> I'm closing the bug
